typescript/sdk/src/token/types.test.ts

```
@@ -1,176 +1,962 @@
+          token: SOME_ADDRESS,
+          owner: SOME_ADDRESS,
+          mailbox: SOME_ADDRESS,
+        },
+      };
+    });
+
+    it('should require token type', () => {
+      expect(WarpRouteDeployConfigSchema.safeParse(config).success).to.be.true;
+
+      //@ts-ignore
+      delete config.arbitrum.type;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)